charts/garden-extension-validator-gcp/charts/runtime/templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels" . | indent 4 }}
+spec:
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.global.replicaCount }}
+  selector:
+    matchLabels:
+{{ include "labels" . | indent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/secret-gardener-extensions-validator-gcp-cert: {{ include (print $.Template.BasePath "/secret-cert.yaml") . | sha256sum }}
+        {{- if .Values.global.kubeconfig }}
+        checksum/gardener-extensions-validator-gcp-kubeconfig: {{ include (print $.Template.BasePath "/secret-kubeconfig.yaml") . | sha256sum }}
+        {{- end }}
+      labels:
+{{ include "labels" . | indent 8 }}
+    spec:
+      {{- if .Values.global.kubeconfig }}
+      automountServiceAccountToken: false
+      {{- else }}
+      serviceAccountName: {{ include "name" . }}
+      {{- end }}
+      containers:
+      - name: {{ include "name" . }}
+        image: {{ include "image" .Values.global.image }}
+        imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+        command:
+        - /gardener-extension-hyper
+        - validator-gcp
+        - --webhook-config-server-port={{ .Values.global.webhookConfig.serverPort }}
+        - --webhook-config-cert-dir=/etc/gardener-extensions-validator-gcp/srv
+        {{- if .Values.global.kubeconfig }}
+        - --kubeconfig=/etc/gardener-extensions-validator-gcp/kubeconfig/kubeconfig
+        {{- end }}
+        ports:
+        - name: webhook-server
+          containerPort: {{ .Values.global.webhookConfig.serverPort }}
+          protocol: TCP
+        livenessProbe:
+          tcpSocket:
+            port: {{ .Values.global.webhookConfig.serverPort }}
+          initialDelaySeconds: 5
+          periodSeconds: 10
+{{- if .Values.global.resources }}
+        resources:
+{{ toYaml .Values.global.resources | nindent 10 }}
+{{- end }}
+        volumeMounts:
+        - name: gardener-extensions-validator-gcp-cert
```

Comment:
Rename everything from `gardener-extensions-validator...` to `gardener-extension-validator`